### ISSUE TITLE
Investigate deploying the application with Solo

### ISSUE THREAD

**Issue by SimiHunjan** [hashgraph/hedera-transaction-tool#1207]
### Problem

The application is not currently deployed with Solo. 

### Solution

Review Solo and identify how the Transaction Tool can be deployed with it.

**Comment by SimiHunjan** [comment]
Yes, this is the Solo project: https://github.com/hashgraph/solo. We want to spin up the backend of the Transaction Tool when a new Solo network is created just like the services are automatically deployed like consensus node, mirror node, etc. This way the backend of the Transaction Tool can automatically be setup for use. The expectation is that our team would contribute a PR to Solo that adds this functionality, I believe this will entail creating a helm chart etc.

**Comment by jbair06** [comment]
Work has begun, see #1424
